### ISSUE TITLE
Rounding errors in get_next_trial()

### ISSUE THREAD

**Issue by yoavnavon** [facebook/Ax#439]
Hi, I'm creating an experiment with the following parameters:

```python
domain = [
    {
         "name": "emb_10",
          "value_type": "float",
          "type": "range",
          "bounds": [-3.07596, 0.32121],
    }
]
ax_client.create_experiment(
                name=experiment_config["exp_name"],
                parameters=domain,
                objective_name=experiment_config["obj_name"],
                parameter_constraints=[],
                outcome_constraints=[],
                minimize=experiment_config["minimize"],
            )
```

But when I generate new points, using `ax_client.get_next_trial()` the proposed trials fall outside of the bounds, probably because of a rounding error. When I try to attach the point I get the following error:
```python
ValueError: 0.3212100000000002 is not a valid value for parameter RangeParameter(name='emb_10', parameter_type=FLOAT, range=[-3.07596, 0.32121])
```

**Comment by adamobeng** [comment]
Hi @yoavnavon! I've seen this happen before, but only with log-scale parameters where the transformation was presumably to blame for the numeric error. Let me look into this.

**Comment by adamobeng** [comment]
It looks like we'd previously figured out a fix for this issue, it should available in the repo in the next couple of days!

**Comment by adamobeng** [comment]
@yoavnavon : our suggested fix is in this commit: https://github.com/facebook/Ax/commit/e4498ac9d54a9f77dc33ed1e28df253661574753

Could you see whether this works for you?

**Comment by lena-kashtelyan** [comment]
It seems that this is likely fixed in master, so marking this accordingly. We will be releasing a stable version including the fix soon.

**Comment by lena-kashtelyan** [comment]
This should now be fixed in latest stable release, 0.1.20.
